test/unit/puppetlabs/services/jruby/jruby_interpreter_test.clj

```diff
@@ -1,64 +0,0 @@
-(ns puppetlabs.services.jruby.jruby-interpreter-test
-  (:require [clojure.test :refer :all]
-            [me.raynes.fs :as fs]
-            [puppetlabs.services.jruby.jruby-testutils :as jruby-testutils]
-            [puppetlabs.services.jruby.jruby-puppet-internal :as jruby-internal]
-            [puppetlabs.puppetserver.testutils :as testutils]
-            [puppetlabs.kitchensink.core :as ks]))
-
-(use-fixtures :once
-              (testutils/with-puppet-conf
-                "./dev-resources/puppetlabs/services/jruby/jruby_interpreter_test/puppet.conf"))
-
-(deftest create-jruby-instance-test
-
-  (testing "Var dir is not required (it will be read from puppet.conf)"
```

Comment:
Are the tests in this block which validate settings being honored from puppet.conf vs. from the tk-config which are being removed still covered elsewhere?